README.md

```diff
@@ -12,6 +12,19 @@ The [`tinywallet`](./tinywallet) package contains a wallet based on the
 Each package may be installed from the [Python Package Index](https://pypi.org/)
 using the [`pip`](https://pip.pypa.io/) command as usual.
 
+## Requirements
+
+To run tinydecred on your machine you will need to set up the following. 
+
+- pip install websocket_client
+- pip install blake256
+- pip install base58
+- pip install PyNaCl
+- pip install appdirs
```

Comment:
You shouldn't install this way. Regular users should just `pip install decred`.

For tinywallet, you can `poetry install` and then `poetry run tinywallet`, but you'll need the fix from #198.